.github/workflows/ci.yml

```diff
@@ -167,3 +168,47 @@ jobs:
         run: cd unpack/espeak-ng-* && make
       - name: re-make check
         run: cd unpack/espeak-ng-* && make check
+  
+  fuzzing:
+
+    runs-on: ubuntu-latest
+    name: Fuzz synth_espeak on ${{ matrix.arch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [x86-32, x86-64]
+        include:
+          - arch: x86-32
+            archdeps: "gcc-multilib g++-multilib libpcaudio-dev:i386 libsonic-dev:i386 libc6-dbg:i386"
+            archconfigflags: "-m32"
+
+          - arch: x86-64
+            archdeps: ""
+            archconfigflags: ''
+    steps:
+    - uses: actions/checkout@v2
+    - name: enable 32bit architecture
+      run: sudo dpkg --add-architecture i386
+      if: matrix.arch == 'x86-32'
+    - name: dependencies
+      run: sudo apt-get update && sudo apt-get install libpcaudio-dev libsonic-dev ronn kramdown clang llvm ${{ matrix.archdeps }}
+    - name: autoconf
+      run: ./autogen.sh ; chmod -x INSTALL m4/*.m4
+    - name: configure
+      run: CC=clang CXX=clang++ CFLAGS="${{ matrix.archconfigflags }}" CXXFLAGS="${{ matrix.archconfigflags }}" ./configure --with-libfuzzer
+    - name: Store the fuzzer config
+      if: ${{ failure() }}
+      uses: actions/upload-artifact@v2
+      with:
+        name: fuzzer-config-${{ matrix.arch }}.log
+        path: tests/fuzzer-config.log
+    - name: make
+      run: make -j
+    - name: Fuzz function syth_espeak()
+      run: FUZZ_VOICE=en tests/fuzzing/synth_fuzzer -seed=1 tests/fuzzing/CORPUS_DIR
```

Comment:
You need to set the `ESPEAK_DATA_PATH` environment variable, see that CI https://github.com/yocvito/espeak-ng/runs/6248639305?check_suite_focus=true fails otherwise